common/src/traits.rs

```diff
@@ -45,3 +48,18 @@ pub trait NFTTrait {
     /// TODO!
     fn is_converted_to_capsule(id: NFTId) -> Option<bool>;
 }
+
+/// Trait that implements basic functionalities related to Ternoa Marketplace
+/// TODO: Expand trait with more useful functions
+pub trait MarketplaceTrait<AccountId> {
```

Comment:
Out of curiosity, is there a conceptual or technical difference between using a generic type or a associated type?